config/routes.rb

```diff
@@ -5,6 +5,7 @@
 
   resource :dashboard, only: [:index]
   resource :updates, only: [:create]
+  resources :nodes, only: [:destroy]
```

Comment:
I totally agree on moving stuff to `NodesController` and being more resources oriented but only after the release. We'll have a sprint to cleanup stuff after the release and I think we should also prioritize this.

In #173 I added the actions to the dashboard controller but it should definitely be in `NodesController`. =/